### PR TITLE
Allie character pick up socket and key binding

### PR DESCRIPTION
**Pull request by Aledria**
Created a socket named "PickupSocket" on the RobotDog_Skeleton

image

Created a socket named "PickupSocket" on the SK_Mannequin

image

Changed the target socket for pick up item action

image

Changed Pickup action keyboard mapping from "p" to "spacebar"

### CODE REVIEW

**Comment by tpower3** [comment]
Would you mind deleting the socket named `hand_r_socket` on the `SK_Mannequin`? It serves no purpose once this PR has merged. Otherwise this looks good. I'll approve once you've indicated that you've made this change.

**Comment by Aledria** [comment]
Good catch! Change is made!

**Review by tpower3 — APPROVED** [review]
LGTM

**Pull request merged**
